elements/pl-order-blocks/pl-order-blocks.css

```diff
@@ -14,8 +14,18 @@ div.pl-order-blocks-bottom {
 
 li.pl-order-blocks-code {
   list-style-type: none;
+  padding: 0px !important;
```

Comment:
why is this needed?

Reply:
The padding changes that made the default format option need to overridden for blocks containing pl-code so they don't get bigger.